src/Components/App/App.js

```diff
@@ -31,25 +31,31 @@ class App extends Component {
     }
   }
 
-  getActivityData = async (event) => {
+  getActivityData = async (name) => {
+    if (name.target) {
+      name = name.target.id
+    }
+    if (name.includes('games')) {
+      name = name.replace(/ games/gi, 'Games')
```

Comment:
Maybe we can think of a way to not use regex here, but for now it does the trick.